weatherman.py

```diff
@@ -0,0 +1,28 @@
+# Import Weatherman class and argument library
+import argparse
+from weather import Weatherman
+
+
+# Arguments that are acceptable in the command line
+parser = argparse.ArgumentParser(description="A weather reporting application")
+parser.add_argument('path',
+                    help="Path to file Extraction")
+parser.add_argument('-e', '--Temp_Humidity', nargs='+',
```

Comment:
Follow python naming conventions for arguments and use proper names.
```suggestion
parser.add_argument('-e', '--temp_humidity', nargs='+',
```

Reply:
change made as suggested.